code/dynamic_programming/minimum_cost_polygon_triangulation/MinimumCostPolygonTriangulation.cpp

```diff
@@ -0,0 +1,71 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <iostream>
+#include <cmath>
+#define MAX 1000000.0
```

Comment:
`const int` is better than a macro. Also, I wouldn't make it global either. I would make it static const inside the function it is used in.